app/models/goal.py

```diff
@@ -2,4 +2,26 @@
 
 
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    tasks = db.relationship("Task", back_populates="goal", lazy=True)
+
+
+    @classmethod
+    def to_dict(cls, self):
+        return {
+            "id": self.id,
+            "title":self.title
+        }
```

Comment:
I don't think this needs to be a class method.  An instance method works just as well.